### PR TITLE
Crow operator

### PR DESCRIPTION
**Pull request by lucianmot**

### CODE REVIEW

File: forest.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import re
 
-TOKENS =   [re.compile(r'(?P<ECHO>echo)'), re.compile(r'(?P<BOOLEAN>true|false)'), re.compile(r'(?P<EQUALS>OvO)'), 
-            re.compile(r'(?P<NOT_EQUAL>XvX)'), re.compile(r'(?P<INTEGER>\d)'), re.compile(r'(?P<STRSTART><<)'), 
-            re.compile(r'(?:<<)(?P<STRING_CONTENT>.+)(?:>>)'), re.compile(r'(?P<STRSTOP>>>)')]
+TOKENS =   [re.compile(r'(?P<ECHO>echo)'), re.compile(r'(?P<BOOLEAN>true|false)'), re.compile(r'(?P<MODULUS>\(\*\)\>)'),
```
Comment (aniasobo) [inline forest.py:3]: 🐧🐧🐧

**Review by aniasobo — APPROVED** [review]

File: forest.py (hunk 1 above)
Comment (ajmpawlik) [reply forest.py:3]: nice!

**Review by ajmpawlik — APPROVED** [review]
Great!

**Review by zepherine2006DEV — APPROVED** [review]
Great!! Please merge!!!!!

**Pull request merged**